### ISSUE TITLE
Mocks for testing

### ISSUE THREAD

**Issue by vrudikov** [kafkajs/confluent-schema-registry#221]
Hi! Is there any possibility to mock registry schema so it can use locally parsed schemas only? This can be useful for testing purposes to avoid Docker usage

**Comment by Akupsmee** [comment]
Yes, this will be useful, i just ran into a usecase

**Comment by Akupsmee** [comment]
This will be useful, i just ran into a usecase

**Comment by Akupsmee** [comment]
Yes, this will be useful, i just came across a case where i needed a mock.

**Comment by davidscuderi-facephi** [comment]
Totally, we need this for unit testing in a clean way
